backend/enhanced_guide_generator.py

```
@@ -0,0 +1,796 @@
+                "day": day_number,
+                "date": current_date.strftime('%Y-%m-%d'),
+                "day_name": current_date.strftime('%A'),
+                "theme": f"Day {day_number}: Luxury Paris Experience",
+                "activities": day_activities,
+                "meals": {
+                    "breakfast": "Hotel breakfast or room service",
+                    "lunch": "Included in activities or recommended restaurants",
+                    "dinner": "Michelin-starred dining experiences"
+                },
+                "transportation": "Private car service available throughout the day",
+                "notes": f"All activities include VIP access and private guides where applicable"
```

Ruff (0.13.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)